### PR TITLE
Update demo notebooks to v2 syntax

### PR DESCRIPTION
**Pull request by jjbrosnan**
All notebooks have V2 syntax with the exception of:

`02 Steam and Batch Together.md`

That notebook is awating Python wrappers for `formatColumns` and `formatColumnWhere`.  Once those are in main, the notebook will be updated and this will be ready for a merge.

### CODE REVIEW

File: demo/web/src/main/notebooks/08 Table Listeners.md [hunk 1]
```diff
@@ -5,19 +5,19 @@ One of Deephaven's defining features is its ability to handle real-time data wit
 To start, let's make a table that updates every two seconds with a random number.
 
 ```python
-from deephaven.TableTools import timeTable
+from deephaven import time_table
 
 import random
 
-table = timeTable('00:00:02').update("Number = (int)(byte)random.randint(1,100)")
+table = time_table('00:00:02').update(["Number = (int)(byte)random.randint(1,100)"])
```
Comment (jakemulf) [inline demo/web/src/main/notebooks/08 Table Listeners.md:12]:
```suggestion
table = time_table('00:00:02').update(["Number = (int)random.randint(1,100)"])
```

File: demo/web/src/main/notebooks/08 Table Listeners.md [hunk 2]
```diff
@@ -50,8 +50,8 @@ What if we had two time tables and wanted to reuse the same listener method? We
 Let's start with our two time tables.
 
 ```python
-table_one = timeTable('00:00:02').update("Number = (int)(byte)random.randint(1,100)")
-table_two = timeTable('00:00:05').update("Number = (int)(byte)random.randint(1,100)")
+table_one = time_table('00:00:02').update(["Number = (int)(byte)random.randint(1,100)"])
+table_two = time_table('00:00:05').update(["Number = (int)(byte)random.randint(1,100)"])
```
Comment (jakemulf) [inline demo/web/src/main/notebooks/08 Table Listeners.md:54]:
```suggestion
table_one = time_table('00:00:02').update(["Number = (int)random.randint(1,100)"])
table_two = time_table('00:00:05').update(["Number = (int)random.randint(1,100)"])
```

File: demo/web/src/main/notebooks/A. Real-time table ops/A1 Create tables and fake data.md [hunk 3]
```diff
@@ -38,13 +36,11 @@ def create_random_table(time_interval, start_time=None):
...
-        table = timeTable(time_interval)
+        table = time_table(time_interval)
     else:
-        table = timeTable(start_time, time_interval)
+        table = time_table(period=time_interval, start_time=start_time)
 
-    return table.update("Number = (int)(byte)random_int()")\
-            .update("Character = (String)random_character()")\
-            .update("Boolean = (boolean)random_boolean()")
+    return table.update(formulas=["Number = (int)(byte)random_int()", "Character = (String)random_character()", "Boolean = (boolean)random_boolean()"])
```
Comment (jakemulf) [inline demo/web/src/main/notebooks/A. Real-time table ops/A1 Create tables and fake data.md:43]:
```suggestion
    return table.update(formulas=["Number = (int)random_int()", "Character = (String)random_character()", "Boolean = (boolean)random_boolean()"])
```

**Review by jakemulf — CHANGES_REQUESTED** [review]
Small syntax updates, there's more for me to go through but I'll finish the review later

**Review by hythloda — APPROVED** [review]

**Pull request merged**